build_projects/shared-build-targets-utils/Utils/Dirs.cs

```diff
@@ -17,7 +17,7 @@ public static class Dirs
 
         public static readonly string Intermediate = Path.Combine(Output, "intermediate");
         public static readonly string PackagesIntermediate = Path.Combine(Output, "packages/intermediate");
-        public static readonly string PackagesNoRID = Path.Combine(RepoRoot, "artifacts", "packages");
+        public static readonly string PackagesNoRID = Path.Combine(RepoRoot, "pkg", "packages", "AzureTransfer", "pkg");
```

Comment:
?? any assets that are the output of the build need to keep going into the `artifacts` directory.

Reply:
This doesn't refer to packages that we've just built, but packages that we've downloaded to re-publish in the finalize step. https://github.com/dotnet/core-setup/blob/4cc6ac5f43f48b3a6fd3ab73763ce9f287c05fa2/build_projects/shared-build-targets-utils/Utils/Dirs.cs#L21 is where we put the produced packages from a build.

Reply:
Sure, how about `PackagesForFinalPublish` ?

Reply:
And this will only include packages built by Core-Setup (although that includes NetCore & NetCore.App)

Reply:
That sounds pretty good, maybe also add Downloaded for `DownloadedPackagesForFinalPublish` to make it super clear they aren't being built on the machine.

Reply:
Oh, and I guess I'm assuming this change is because of the switch to use the BuildToolsy download, rather than a CLI C# build project task, and it would take more effort to move the download location to `artifacts/something`. In general I agree this would belong in `artifacts/` otherwise.

Reply:
I don't understand why we are putting assets that the product produces into the source tree under the `pkg` folder.  Assets that are produced during the build should go in the `artifacts` folder.  Even if they are downloaded from somewhere else.

Reply:
Essentially this is the same as the `packages` folder in CoreFx/CoreClr, except it exists one level down the tree in `pkg`. This directory already exists before these changes, as it's where `init-tools.cmd` puts the packages it restores.